lib/dat-worker-pool/worker.rb

```diff
@@ -87,6 +87,10 @@ def raise(*args)
         @thread.raise(*args) if self.thread_alive?
       end
 
+      # Helpers - TODO make these private once callbacks are instance evald
+      def params; @runner.worker_params; end
```

Comment:
@jcredding btw I saw how the tests took care of this by using a 'locked array' for one param they had multiple workers write to.  I'm cool letting them be responsible for making the param values thread-safe as long as the base read only case doesn't need any special thread handling.